queries/omen/markets.ts

```diff
@@ -166,7 +166,7 @@ const getMarketsQuery = (
         ${params.title_contains_nocase ? 'title_contains_nocase: $title_contains_nocase' : ''}
         ${params.id ? 'id: $id' : ''}
         ${params.creator_in ? 'creator_in: $creator_in' : ''}
-        ${params.category_contains ? 'category_contains: $category_contains' : ''}
+        ${params.category_contains ? 'category_contains: $category_contains' : 'category_not_contains: "test"'}
```

Comment:
I think we should reserve this line only for `category_contains`.

Let's add `category_not_contains` in another line.

Will simplify how we read too.